docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -227,4 +227,47 @@ InferenceOptimizer.quantize(model,
                             approach='static',
                             max_trials=10,
                             ):
-```
\ No newline at end of file
+```
+
+## Context manager
+BigDL-Nano provides ``InferenceOptimizer.get_context(model=...)`` API to enable automatic context management for PyTorch inference. Only one line of code change, then Nano will automatically provide suitable context management for each accelerated model, it usually contains part of or all of following three types of context manager:
+
+1. torch.no_grad() to disable gradients, which will be used for all model
+   
+2. torch.cpu.amp.autocast(dtype=torch.bfloat16) to run in mixed precision, which will be provided for bf16 related model
+   
+3. torch.set_num_threads() to control thread number, which will be used only if you specify thread_num when `trace/quantize/optimize`
+
+For model accelerated by ``InferenceOptimizer.trace``, usage now looks like below codes, here we just take `ipex` for example:
+```python
+from bigdl.nano.pytorch import InferenceOptimizer
+ipex_model = InferenceOptimizer.trace(model,
+                                      use_ipex=True,
+                                      thread_num=4)
+
+with InferenceOptimizer.get_context(ipex_model):
+    output = ipex_model(x)
+    assert torch.get_num_threads() == 4  # this line just to let you know Nano has provided thread control automatically : )
+```
+
+For ``InferenceOptimizer.quantize`` and ``InferenceOptimizer.optimize``, usage is the same.
+
+``InferenceOptimizer.get_context(model=...)`` can be used for muiti models. If you have a model pipeline, you can also get a common context manager by passing multi models to `get_context`.
```

Comment:
As "multi" is a prefix, seems like we should add a hyphen between it and the word after it. That is "multi-models", "multi-context". Similar problems below.